### PR TITLE
Add a simple way to do a pure simulation of an obsid

### PR DESCRIPTION
**Pull request by taldcroft**
From the current validation notebooks it seems to take many lines of code to get annie to do anything.  This PR is meant to enable doing a pure simulation (NO flight telemetry) in one line of code.  It could also be the start of regression tests (runnable with `python setup.py test`).

### CODE REVIEW

**Comment by taldcroft** [comment]
This includes a couple of unrelated changes (sorry), one a fix for the `attitude` import and removing the `print` statements.  The latter is probably not useful at this point.

**Comment by malgosias** [comment]
At the moment test_track_guide_stars works with img sizes 4|6|8. Catalogs fetched from starcheck have larger cat['halfw']. Possible fixes include:
- in ``test_track_guide_stars`` add ``cat['sz'] = 8``, ``cat['halfw'] = 20`` after fetching the starcheck catalog, or
- in ``aca.py`` in ``calc_flight_background`` replace ``if sdr.image.shape != (8, 8):`` with ``if sdr.image.shape == (6, 6):``, or
- move the  functionality of ``if sdr.image.shape != (8, 8):`` statement one level up to from ``calc_flight_background`` to ``calc_bgd`` and do the following check: if it is not a simulation, pop bgdavg value from telemetry, else the current content of ``calc_bgd``.

**Comment by taldcroft** [comment]
> in test_track_guide_stars add cat['sz'] = 8, cat['halfw'] = 20 after fetching the starcheck catalog, or

This comment confuses me since `cat['halfw']` is only related to searching, so it should have no impact on guide star tracking simulation.

**Comment by jeanconn** [comment]
Tom beat me, but for the first bullet, not sure why you'd hardcode cat['sz'] to be 8 instead of converting  the starcheck 'sz' to be 8 or 6 from the '8x8' or '6x6' read from the starcheck catalog.

**Comment by malgosias** [comment]
I hardcoded cat['sz'] to be 8 for simulation because we always simulate internal 8x8 readout windows, independent of 6x6 or 8x8 tag in starcheck catalog. (If shining telemetry the sz is determined by the telemetry image, so it can be different from 8.)

Halfw is currently used to predict row0, col0, see ``star_data_record_prepare_read`` in ``aca.py``. This is inherited from an early version of the simulator. This should be changed to always use ``sz`` instead of ``halfw`` (on my list). I agree that using ``halfw`` along with ``sz`` is redundant and confusing; ``sz`` conveys all the information needed in the guide phase.

**Comment by taldcroft** [comment]
Oops that's a bug in the original...

**Comment by taldcroft** [comment]
I'm working on this now BTW.

**Comment by taldcroft** [comment]
Superceded by #6.

File: annie/annie.py [hunk 1]
```diff
@@ -26,25 +18,27 @@
 telem = Telem()
```
Comment (jeanconn) [inline annie/annie.py]: Regarding the API for test_track_guide_stars, could these annie initializers
```
    ccd = CCD()
    aca = ACA()
    sky = Sky()
    pcad = PCAD()
    clock = Clock()
    telem = Telem()
```
just live inside the test methods (like test_track_guide_stars)?  Or should there just be a reset method at top level to reinitialize them or some such?  I don't fully understand scope.

**Pull request closed without merging**